tutorials/3_RunningBundleOfRuns.md

```diff
@@ -3,36 +3,56 @@ Running more than one REMIND scenario
+
+<img src="figures/scenario_config.png" alt="Example for a scenario_config of REMIND" width="100%" />
+<p class="caption">
+Example for a scenario_config of REMIND
+</p>
+
+Those two columns are mandatory and usually placed at the beginning:
+
+* `title` labels that run. It contains a unique identifier for each run, which must not contain a `.` and must not end with a `_`. The more runs you will have, the more it will be important that you label them in a way such that you easily remember the specific settings you chose for this run.
+* `start` is a boolean switch that lets you choose whether or not you would like to start this run once you submit this config file to the modeling routine. It often makes sense to keep some runs in the csv file to remember their configurations for the next time although you do not want to run them now and therefore switch them off. You do this by setting `start` to 0.
+
+Further columns are the configurations that you can choose for the specific runs.
+They may contain values for parameters such as `cm_rcp_scen` and module realizations such as `exponential` for [`./module/carbonprice/`](../modules/45_carbonprice). They overwrite the default defined and explained in [`./config/default.cfg`](../develop/config/default.cfg) by the respective cell value for each run. If you leave a cell empty or if no column exists for a setting, the default value is used.
 
-Viewed as a spreadsheet, the first row contains the column titles, while subsequent rows each contain a remind run.
+Note that the cell need not contain only a single value, but for example module realization [`47_regipol/regiCarbonPrice`](../develop/modules/47_regipol/regiCarbonPrice) allows to specify in the parameter `cm_regiCO2target` to enter comma separated values `2020.2050.USA.year.netGHG 1, 2020.2050.EUR.year.netGHG 1` to specify emission goals for multiple regions.
 
-The first two columns are mandatory and usually placed at the beginning.
+To compare a `scenario_config*.csv` file to the current default configuration, you can run `Rscript -e "remind2::colorScenConf()"` in your remind directory and select the file you are interested in. [`colorScenConf()`](https://github.com/pik-piam/remind2/blob/master/R/colorScenConf.R) produces a file ending with `_colorful.xlsx` in the same directory and provides you with information how to interpret the colors within.
 
-* `title` contains a unique identifier for each run, which must not contain a `.` and must not end with a `_`.
-* `start` is a boolean switch, either 1 if the run is to be executed once run with `start.R`, or 0 if not.
+An important feature of scenario_config files is the possibility to execute runs which build on each other.
+An example is a base run without climate policies and a policy run with carbon taxes.
+Such a "subsequent run" has to wait for the run it builds on to be finished.
```

Comment:
maybe add here the most common example for subsequent runs where the first time steps until `startyear` are used form the first scenario.

Reply:
I did